### PR TITLE
io.github.Snaggly.PS2_Pnacher

### PR DESCRIPTION
**Pull request by Snaggly**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions (i think).
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project.
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned.

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build com.snaggly.ps2pnach

File: com.snaggly.ps2pnach.yml [hunk 1]
```diff
@@ -0,0 +1,132 @@
+app-id: com.snaggly.ps2pnach
+runtime: org.gnome.Platform
+runtime-version: '3.38'
```
Comment (barthalion) [inline com.snaggly.ps2pnach.yml:3]:
```suggestion
runtime-version: '40'
```

File: com.snaggly.ps2pnach.yml [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+  - '/share/gtk-doc'
+  - '/share/info'
+  - '/share/man'
+  - '/share/pkgconfig'
+  - '/lib/cmake'
+  - '*.la'
+  - '*.a'
+modules:
+  - name: ps2pnacher
+    buildsystem: cmake
```
Comment (barthalion) [inline com.snaggly.ps2pnach.yml:24]:
```suggestion
    buildsystem: cmake-ninja
```

File: com.snaggly.ps2pnach.yml [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    config-opts:
+    - -DCMAKE_BUILD_TYPE=Release
+    - -DCMAKE_INSTALL_PREFIX=/app
```
Comment (barthalion) [inline com.snaggly.ps2pnach.yml:27]:
```suggestion
```

File: com.snaggly.ps2pnach.yml [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+    sources:
+      - type: git
+        url: 'https://github.com/Snaggly/PS2_Pnacher.git'
+        branch: 'main'
```
Comment (barthalion) [inline com.snaggly.ps2pnach.yml:31]:
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,2 @@
+# PS2 Pnacher
```
Comment (barthalion) [inline README.md:1]: No need to have that file.

**Comment by Snaggly** [comment]
bot, build com.snaggly.ps2pnach

**Comment by Snaggly** [comment]
bot, build com.snaggly.ps2pnach

**Comment by barthalion** [comment]
Actually… As you don't own `snaggly.com`, it really should be something along `io.github.Snaggly.PS2_Pnacher`. The idea behind `I own the domain used in the application ID` is to have ownership bound to the domain.

**Comment by Snaggly** [comment]
Right... I have changed the Id now. However the branch still holds the old Id. Should I open a new pr from a new branch with the updated AppId or keep as is?

**Comment by Snaggly** [comment]
bot, build io.github.Snaggly.PS2_Pnacher

**Comment by hfiguiere** [comment]
You can edit the title of the pull request. it's cosmetic (but useful) (the branch name doesn't matter at all)

**Pull request closed without merging**